### ISSUE TITLE
Tags in metadata not recognized by plugin

### ISSUE THREAD

**Issue by Unimp0rtance** [karstenpedersen/obsidian-advanced-random-note#1]
```
---
time: 2023-08-09 00:00 Wednesday
title: test
tags: 
- permanent
---
```

This is the general format of my notes.
This will be recognized by obsidian as a Tag, but it's  prompt "Found zero notes" by Advanced random note.

I have observed that obsidian may improve the metadata area in the near future. 
If you plan to support it after the update, it also very grateful!

**Comment by karstenpedersen** [comment]
Hi @Unimp0rtance thanks for the constructive feedback :)

I have changed the code to also include frontmatter tags and published it as version `0.0.7`.

If you have time, could you also check if it works and report back? That would help a lot.

Thanks

**Comment by karstenpedersen** [comment]
I have updated the plugin to version [0.0.8](https://github.com/karstenpedersen/obsidian-advanced-random-note/releases/tag/0.0.8) which enables queries to use the [Dataview](https://github.com/blacksmithgu/obsidian-dataview) plugin and regular expressions.

I will create a new issue with this problem as I have to refactor a bit of code. 

Thanks for the constructive feedback :)
